pycaching/geocaching.py

```diff
@@ -367,3 +376,34 @@ def post_log(self, wp, text, type=LogType.found_it, date=None):
             date = datetime.date.today()
         l = Log(type=type, text=text, visited=date)
         self.get_cache(wp).post_log(l)
+
+    def _cache_from_guid(self, guid):
+        logging.info('Loading cache with GUID {!r}'.format(guid))
+        print_page = self._request(Cache._urls["print_page"], params={"guid": guid})
+        return Cache._from_print_page(self, guid, print_page)
+
+    def _my_logs(self, log_type, limit):
+        logging.info("Getting {} of my logs of type {}".format(limit, log_type))
+        cache_table = self._request(self._urls['my_logs'].format(lt=log_type)).find(class_='Table')
+        if cache_table is None:  # no finds on the account
+            return
+        cache_table = cache_table.tbody
+
+        yielded = 0
+        for row in cache_table.find_all('tr'):
+            link = row.find(class_='ImageLink')['href']
+            guid = parse_qs(urlparse(link).query)['guid'][0]
+
+            if yielded >= limit:
+                break
+
+            yield self.get_cache(guid=guid)
+            yielded += 1
+
+    def my_finds(self, limit=float('inf')):
```

Comment:
Can you clarify what you mean by this? What is a shortcut in this context?

Reply:
Python doesn't have private methods, but I've followed convention by naming the method `_my_logs`. The leading underscore signifies a "private" method (though of course nothing enforces the privacy). This seems to be in line with other methods in pycaching, for example:

https://github.com/tomasbedrich/pycaching/blob/d1e42d730edc48b91799182d5b97700628697fc6/pycaching/geocaching.py#L269

Reply:
Oops, I misread your second point and gave an incorrect response. I deleted it.

I kept it private because I don't think the other log types are really interesting or useful to "end users." Here's the full list available:

> Archive, Attended, Didn't find it, Enable listing, Found it, Needs archived, Needs maintenance, Owner maintenance, Post reviewer note, Submit for review, Temporarily disable listing, Update coordinates, Will attend, Write note, All Logs

If you think I should "de-private" that method, I'd be happy to. Let me know! 🙂

Reply:
Yes please. I can imagine use cases where this can be useful.

Also, adding a note about new methods to readme would be nice. ;)

Reply:
I believe e8f70969c9f5fd1a8f4a1a93690dd8e553f8a3c9 takes care of this.